tests/org.jboss.tools.easymport.ui.bot.test/pom.xml

```diff
@@ -38,11 +38,6 @@
 							<artifactId>org.eclipse.jdt.feature.group</artifactId>
 							<version>0.0.0</version>
 						</dependency>
-						<dependency>
```

Comment:
You do not need the dep anymore? Or it was useless for some time already?

Reply:
@odockal this has moved to egit - I have seen some email about this.